google/cloud/spanner/testing/cleanup_stale_instances.cc

```diff
@@ -41,6 +41,19 @@ std::string CutoffDate() {
 }  // namespace
 
 Status CleanupStaleInstances(Project const& project) {
+  spanner_admin::InstanceAdminClient instance_admin_client(
+      spanner_admin::MakeInstanceAdminConnection());
+  spanner_admin::DatabaseAdminClient database_admin_client(
+      spanner_admin::MakeDatabaseAdminConnection());
+  return CleanupStaleInstances(project, instance_admin_client,
+                               database_admin_client);
```

Comment:
`std::move` the clients, and then accept them by value (et seq).

Reply:
Done.